components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Deploy/IoTHubMetricsCollectorDeployment.cs

```diff
@@ -108,11 +106,20 @@ public sealed class IoTHubMetricsCollectorDeployment : IHostProcess {
                         ""type"": ""docker"",
                         ""version"": ""1.0"",
                         ""env"": {
-                            ""AzMonWorkspaceId"": {
+                            ""UploadTarget"": {
+                                ""value"": ""AzureMonitor""
+                            },
+                            ""LogAnalyticsWorkspaceId"": {
                                 ""value"": """ + _config.LogWorkspaceId + @"""
                             },
-                            ""AzMonWorkspaceKey"": {
+                            ""LogAnalyticsSharedKey"": {
                                 ""value"": """ + _config.LogWorkspaceKey + @"""
+                            },
+                            ""ResourceId"": {
+                                ""value"": """ + _config.IoTHubResourceId + @"""
+                            },
+                            ""MetricsEndpointsCSV"": {
```

Comment:
Should we add discovery module as well?  Does this value support wildcards?

Reply:
For some reason the discovery was not part of this in the first place. That's the only reason I did not add it. 
I did not find any reference in the documentation on supporting wildcards.